### PR TITLE
Feature/docstring methods

### PR DESCRIPTION
**Pull request by eruizgar91**
## Description
Add doc strings in the methods that were missing.
## Is this PR related with an open issue?

Related to Issue #154 

## Types of changes

- [] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [] Follows the code style of this project.
- [ ] Tests Cover Changes
- [x] Documentation

### CODE REVIEW

File: squid_py/config.py [hunk 1]
```diff
@@ -56,11 +54,16 @@
...
-    """
+    """Class to manage the squid-py configuration."""
 
     def __init__(self, filename=None, options_dict=None, **kwargs):
+        """
+        Initialize Config class.
+
+        :param filename: Path of the config file, str.
+        :param options_dict: Python dict with the config, dict.
+        :param kwargs: Additional args.
```
Comment (ttmc) [inline squid_py/config.py]: Maybe you could say something about what optional args could be included (e.g. `logger`, `text`), their default values, their type, etc.

File: squid_py/ddo/ddo.py [hunk 2]
```diff
@@ -51,11 +46,22 @@ def __init__(self, did='', json_text=None, json_filename=None, created=None, dic
...
+        """
         self._public_keys.append(public_key)
 
     def add_authentication(self, key_id, authentication_type=None):
-        """add a authentication public key id and type to the list of authentications"""
+        """
+        Add a authentication public key id and type to the list of authentications.
+
+        :param key_id: Key id, str
+        :param authentication_type:
```
Comment (ttmc) [inline squid_py/ddo/ddo.py]: Could you add a description and type for this parameter (and any others that are missing one)?

File: squid_py/ddo/ddo.py [hunk 3]
```diff
@@ -72,8 +78,15 @@ def add_authentication(self, key_id, authentication_type=None):
...
     def add_signature(self, public_key_store_type=PUBLIC_KEY_STORE_TYPE_PEM, is_embedded=False):
-        """add a signature with a public key and authentication entry for validating this DDO
-        returns the private key as part of the private/public key pair"""
+        """
+        Add signature.
+
+        Add a signature with a public key and authentication entry for validating this DDO
+        returns the private key as part of the private/public key pair
+        :param public_key_store_type
+        :param is_embedded bool
```
Comment (ttmc) [inline squid_py/ddo/ddo.py]: There seems to be some syntax errors here: no colon `:` after the name of the parameter. (Parameter descriptions are also missing.)

File: squid_py/didresolver.py [hunk 4]
```diff
@@ -34,23 +29,20 @@
...
 
 class DIDResolved:
-    """
-    Class that handles the resolved DID information
-    """
+    """Class that handles the resolved DID information."""
 
     def __init__(self):
-        """init the object with an empty set of hops"""
+        """Init the object with an empty set of hops."""
```
Comment (ttmc) [inline squid_py/didresolver.py]: hops?

File: squid_py/ocean/ocean.py [hunk 5]
```diff
@@ -524,10 +521,12 @@ def resolve_did(self, did):
...
     def _encrypt_metadata_content_urls(self, did, data):
         """
-        encrypt string data using the DID as an secret store id,
+        Encrypt urls.
+
+        Encrypt string data using the DID as an secret store id,
         if secret store is enabled then return the result from secret store encryption
 
-        return None for no encryption performed
+        :return None for no encryption performed
```
Comment (ttmc) [inline squid_py/ocean/ocean.py]:
```suggestion
        :return: None for no encryption performed
```

File: tests/test_didresolver.py [hunk 6]
```diff
@@ -1,94 +1,26 @@
...
-
 from squid_py.exceptions import (
     OceanDIDCircularReference,
     OceanDIDNotFound,
 )
 
 logger = logging.getLogger()
 
-"""
-def test_did_resolver_raw_test():
```
Comment (ttmc) [inline tests/test_didresolver.py]: Maybe you could say something about why you removed this test, i.e. for the record. You could do that by replying to this comment.

Comment (eruizgar91) [reply tests/test_didresolver.py]: I think this test was out of date. During this week we are going continue refactoring the did-resolver and adding new tests.

File: squid_py/didresolver.py (hunk 4 above)
Comment (eruizgar91) [reply squid_py/didresolver.py]: This parameter is about the number of hops to find the did. In the refactoring we are going to simplify this resolver and we are not going to need to expose this parameter.

**Pull request merged**